src/StackExchange.Redis/RedisDatabase.cs

```diff
@@ -2945,6 +2945,9 @@ public Task<long> StringLengthAsync(RedisKey key, CommandFlags flags = CommandFl
             return ExecuteAsync(msg, ResultProcessor.Int64);
         }
 
+        // Backwards compatibility overloads:
+        public bool StringSet(RedisKey key, RedisValue value, TimeSpan? expiry, When when) =>
+            StringSet(key, value, expiry, false, when, CommandFlags.None);
```

Comment:
can't we just make `flags` on the overload below option with a default?

Reply:
Unfortunately I don't think so - the problem is then usage of `RedisKey key, RedisValue value, TimeSpan? expiry, When when` becomes ambiguous between the 2 methods. This was my first play based on your suggestion and my thoughts too, but couldn't make it work.